projects/ion/src/lib/input/input.component.html

```diff
@@ -29,7 +29,7 @@
       [maxlength]="maxLength"
     />
 
-    <span class="text-counter" *ngIf="maxLength">
+    <span class="text-counter" *ngIf="maxLength && value">
       {{ value.length }}/{{ maxLength }}
```

Comment:
```suggestion
    <span class="text-counter" *ngIf="maxLength">
      {{ value ? value.length : 0 }}/{{ maxLength }}
```
Wouldn't it be better to return the value 0 if value?

Reply:
![image](https://github.com/Brisanet/ion/assets/6165180/3562f5b2-a5d8-4c0a-b34b-c80cf69f7af5)

bikeshedding

Reply:
![image](https://github.com/Brisanet/ion/assets/138066684/dcfaecc6-e208-4ba8-936b-8adcefb93d19)
@pedro-martins-brisa suggestion is fine.